tensilelite/Tensile/TensileCreateLibrary.py

```diff
@@ -1268,6 +1268,8 @@ def splitExtraParameters(par):
   argParser.add_argument("--package-library",        dest="PackageLibrary",    action="store_true", default=False)
   argParser.add_argument("--embed-library",          dest="EmbedLibrary",
                          help="Embed (new) library files into static variables.  Specify the name of the library.")
+  argParser.add_argument("--validate-matrix-instructions", dest="ValidateMatrixInstructions", action="store_true",
```

Comment:
Shouldn't this be set to `action="store_false"` because we don't want to run the validation for TensileCreateLibrary?

Reply:
Please refer to the docs here:

https://docs.python.org/3/library/argparse.html

the semantics of store_<true/false> is what to do if the optional argument is encountered. In this case, if we encounter `--validate-matrix-instructions` we want to store the value true if not encountered it defaults to the opposite which is false. For the case of TCL we will default to false because we don't actually use this state for TCL.